src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -410,7 +413,7 @@ public Worker(
                 config.getRetryGetRecordsInSeconds(),
                 config.getMaxGetRecordsThreadPool(),
                 DEFAULT_WORKER_STATE_CHANGE_LISTENER,
-                DEFAULT_LEASE_CLEANUP_VALIDATOR );
+                new KinesisShardSyncer(DEFAULT_LEASE_CLEANUP_VALIDATOR));
```

Comment:
GitHub code organization is kinda messed up. The public constructor hasn't changed - only the initialization within the constructor has changed (which invokes a package-private constructor). This will not break user applications.